### DISCUSSION TITLE
why I get this UserWarning??

### DISCUSSION THREAD

**Discussion by omkarade** [coqui-ai/TTS#1770] [General Q&A]
--> STEP: 0/8 -- GLOBAL_STEP: 0
     | > decoder_loss: 62.89053  (62.89053)
     | > postnet_loss: 64.89718  (64.89718)
     | > stopnet_loss: 0.83731  (0.83731)
     | > loss: 32.78424  (32.78424)
     | > align_error: 0.95189  (0.95189)
     | > grad_norm: 4.09858  (4.09858)
     | > current_lr: 0.00000 
     | > step_time: 108.88960  (108.88960)
     | > loader_time: 10.58780  (10.58783)

/content/TTS/TTS/tts/models/tacotron2.py:333: UserWarning: __floordiv__ is deprecated, and its behavior will change in a future version of pytorch. It currently rounds toward 0 (like the 'trunc' function NOT 'floor'). This results in incorrect rounding for negative values. To keep the current behavior, use torch.div(a, b, rounding_mode='trunc'), or for actual floor division, use torch.div(a, b, rounding_mode='floor').
  ) // self.decoder.r

**Comment by p0p4k** [comment]
My dear friend, you get this message cause you refuse to open your eyes and read what is in front of you. Try it, it might help you understand things better instead of posting a new discussion for every minor inconveniences you encounter in your programming life.
